FitVid/src/Components/Video/Video.js

```diff
@@ -1,26 +1,37 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import {
   FiMoreVertical,
   MdWatchLater,
   RiPlayListAddFill,
   AiFillLike,
   AiOutlineLike,
+  MdOutlineWatchLater,
 } from "../../Assets/icons";
 import {
   ReactPlayer,
   useLiked,
+  useWatchLater,
   findIfVideoExistsInArray,
   addToLikedVideos,
```

Comment:
Suggestion : Good the you are using React Player. You can use onPlay Attribute provided by React Player to add the videos in history